diff --git a/public/fpdfview.h b/public/fpdfview.h
index b374088b4..b1c896e26 100644
--- a/public/fpdfview.h
+++ b/public/fpdfview.h
@@ -214,7 +214,7 @@ typedef int FPDF_OBJECT_TYPE;
 #endif  // defined(FPDF_IMPLEMENTATION)
 #else
 #if defined(FPDF_IMPLEMENTATION)
-#define FPDF_EXPORT __attribute__((visibility("default")))
+#define FPDF_EXPORT __attribute__((visibility("default"))) __attribute__((used))
 #else
 #define FPDF_EXPORT
 #endif  // defined(FPDF_IMPLEMENTATION)
