diff --git a/core/fxge/BUILD.gn b/core/fxge/BUILD.gn
index 215a63b14..357d96735 100644
--- a/core/fxge/BUILD.gn
+++ b/core/fxge/BUILD.gn
@@ -164,7 +164,7 @@ source_set("fxge") {
     sources += [ "linux/fx_linux_impl.cpp" ]
   }
 
-  if (is_mac) {
+  if (is_mac || is_ios) {
     sources += [
       "apple/fx_apple_impl.cpp",
       "apple/fx_apple_platform.cpp",
diff --git a/public/fpdfview.h b/public/fpdfview.h
index b374088b4..b1c896e26 100644
--- a/public/fpdfview.h
+++ b/public/fpdfview.h
@@ -214,7 +214,7 @@ typedef int FPDF_OBJECT_TYPE;
 #endif  // defined(FPDF_IMPLEMENTATION)
 #else
 #if defined(FPDF_IMPLEMENTATION)
-#define FPDF_EXPORT __attribute__((visibility("default")))
+#define FPDF_EXPORT __attribute__((visibility("default"))) __attribute__((used))
 #else
 #define FPDF_EXPORT
 #endif  // defined(FPDF_IMPLEMENTATION)
diff --git a/testing/test.gni b/testing/test.gni
index 6ad2c2d4a..bae5117d8 100644
--- a/testing/test.gni
+++ b/testing/test.gni
@@ -207,7 +207,7 @@ template("test") {
       }
 
       _bundle_id_suffix = target_name
-      if (ios_automatically_manage_certs) {
+      if (true) {
         # Use the same bundle identifier for all unit tests when managing
         # certificates automatically as the number of free certs is limited.
         _bundle_id_suffix = "generic-unit-test"
